### ISSUE TITLE
Please make a gulp version as well

### ISSUE THREAD

**Issue by kristianmandrup** [AppGyver/grunt-steroids#24]
I believe most serious developers have moved away from Grunt by now!?

**Comment by harsaharsa** [comment]
Yup, Gulp is deliciously fresh. Improvements to the "compile" process are on the backlog, but the current implementation works error-free, so we're focusing on other issues at the moment.

**Comment by leefernandes** [comment]
+1

**Comment by sgarbesi** [comment]
+1
